File: samples/format/bad-format.ts
// DELIBERATELY MISFORMATTED. `prettier --check` must reject this file.
//
// Nothing here is a lint or type error — the point is that the FORMATTER is the
// only thing with an opinion about it. See README.md in this directory.
//
// Ignored by .prettierignore so `prettier --write` on the repo cannot quietly
// repair the fixture.

export  const  greeting   =    "hello"

export function  add( a:number,b :number ) : number
{
      return a+b
}

export const items = [
  1,2,
      3,
  4 ]

File: samples/format/bad_format.py
"""DELIBERATELY MISFORMATTED. `ruff format --check` must reject this file.

Nothing here is a lint error — the formatter is the only thing with an opinion
about it. See README.md in this directory.

This file is not covered by `ruff check`; `samples/python` is the lint fixture
and this directory is deliberately outside it, so the finding manifests cannot
move when the formatting fixtures change.
"""


def add(a,b) :
    return   a+b


VALUES = [ 1,2,
        3, 4 ]


class  Thing :
    def __init__( self,name ):
        self.name=name
